monitoring/statuscake/resources.tf

```diff
@@ -21,6 +21,22 @@ resource "statuscake_uptime_check" "main" {
   }
 }
 
+resource "statuscake_heartbeat_check" "main" {
+  name           = var.name
+  period         = var.period
+  contact_groups = var.contact_groups
+  tags           = var.tags
+}
+
+output "example_heartbeat_check_id" {
+  value = statuscake_heartbeat_check.main.id
+}
+
+output "example_heartbeat_check_url" {
+  value = statuscake_heartbeat_check.main.check_url
```

Comment:
Since we want a list of heartbeats, we need to be able to retrieve the url for each name. Can this be a map name => check_url instead?